src/foundry/client/pixi/core/interaction/ping.d.mts

```diff
@@ -55,4 +32,44 @@ declare global {
      */
     protected _animateFrame(dt: number, animation: CanvasAnimationData): void;
   }
+
+  namespace Ping {
+    interface Any extends AnyPing {}
+    type AnyConstructor = typeof AnyPing;
+
+    /** @internal */
+    type _Options = IntentionalPartial<{
+      /**
+       * The duration of the animation in milliseconds.
+       * @defaultValue `900`
+       * @remarks Can't be `null` or `undefined` because `options` is `mergeObject`ed with an object with this key
```

Comment:
This isn't good enough reason in of itself. For example in some data fields you see (basically) `mergeObject(defaults, options)` and `defaults` includes something like `initial: []`. However it's perfectly reasonable to set `initial` to `undefined` or `null` or so on.

Basically you have to measure if it seems intended that you can't have explicit null/undefined.